<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>861</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btn_calc">
       <property name="text">
        <string>Calculator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_Calendar">
       <property name="text">
        <string>Calendar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_FileSystem">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>FileSystem</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_TextEditor">
       <property name="text">
        <string>Text Editor</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>861</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btn_Memory_Stimulator">
       <property name="text">
        <string>Memory Stimulator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_TaskManager">
       <property name="text">
        <string>Task Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_Schedule_Stimulator">
       <property name="text">
        <string>Schedule Stimulator</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionShutdown"/>
   </widget>
   <addaction name="menuSystem"/>
  </widget>
  <action name="actionShutdown">
   <property name="text">
    <string>Shutdown</string>
   </property>
  </action>
  <action name="actionSleep">
   <property name="text">
    <string>Sleep</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
